service/src/main/java/uk/nhs/adaptors/gp2gp/repositorys/EhrExtractStatusRepository.java

```diff
@@ -1,8 +1,8 @@
-package uk.nhs.adaptors.gp2gp.repository;
+package uk.nhs.adaptors.gp2gp.repositorys;
```

Comment:
The package names models, repositorys (a misspelling), etc go against our "package by feature" convention. Fine for now, I'll refactor in the further PR.